common-schema.json

```diff
@@ -3,12 +3,17 @@
   "id": "https://fiware.github.io/dataModels/common-schema.json",
   "title": "Common definitions for FIWARE Harmonized Data Models",
   "definitions": {
+    "EntityIdentiferType": {
+      "type": "string",
+      "minLenght": 1,
+      "maxLength": 256,
+      "pattern": "^[\\w\\-:\\.,;\\(\\)/\\{\\}]+$"
```

Comment:
@jmcanterafonseca offline confirms that is the second case, so the regex should also forbid the following chars:


*    <
*    >
*    "
*    '
*    =
*    ;
    (
    )